crates/python_utilities/src/ast_printer.rs

```diff
@@ -31,15 +33,15 @@ impl<'a> WritingBuffer<'a> {
     }
 
     pub fn deindent(&mut self) -> &mut Self {
-        self.indent += 1;
+        self.indent -= 1;
         self
     }
 
     pub fn push(&mut self, other: Cow<'a, str>) {
         if !self.in_line {
             self.buf.push(Cow::Borrowed("\n"));
             for _ in 0..self.indent {
-                self.buf.push(Cow::Borrowed("  "))
+                self.buf.push(Cow::Borrowed("    "))
```

Comment:
I think we could do one of these above and clone it:

```rust
let i_str = Cow::Borrowed("    ");
for _ in 0..self.indent {
  self.buf.push(i_str.clone());
}
```

Reply:
Addressed in 64150be9f93ffc954a02cfebda6f65460d294c17